def perform_restore(self):
    '\n    Used to perform the restore process\n    :return:\n    '
    try:
        print (('Extracting the archive ' + self.zip_folder_path) + self.zipname)
        unzip((self.zip_folder_path + self.zipname), '/')
    except Exception as e:
        sys.exit((((('Could not extract the zipfile ' + self.zip_folder_path) + self.zipname) + ' Details: ') + str(e)))
